projects/go-lib/src/lib/components/go-action-sheet/go-panel/go-panel.component.ts

```diff
@@ -15,16 +16,21 @@ export class GoPanelComponent {
 
   @Output() action: EventEmitter<void> = new EventEmitter<void>();
 
-  constructor() { }
+  constructor(@Inject(GoActionSheetComponent) private parent: GoActionSheetComponent) { }
```

Comment:
Is this the same as just `private parent: GoActionSheetComponent`. I think the only reason you would need something before the private here is if it's an optional parameter, which in this case it's not. This article talks about when @Inject is necessary vs when it's not 🤷‍♂https://ultimatecourses.com/blog/angular-dependency-injection

Reply:
Yeah, you're right. Forgot you could just do that. I switch it around.